mne/beamformer/_dics.py

```diff
@@ -437,35 +992,97 @@ def tf_dics(epochs, forward, noise_csds, tmin, tmax, tstep, win_lengths,
     win_lengths : list of float
         Time window lengths in seconds. One time window length should be
         provided for each frequency bin.
+    csd_mode : 'fourier' | 'multitaper' | 'cwt_morlet'
+        Spectrum estimation mode. Defaults to 'fourier'.
     freq_bins : list of tuples of float
         Start and end point of frequency bins of interest.
-    subtract_evoked : bool
-        If True, subtract the averaged evoked response prior to computing the
-        tf source grid.
-    mode : str
-        Spectrum estimation mode can be either: 'multitaper' or 'fourier'.
+        Only used in 'multitaper' or 'fourier' mode. For 'cwt_morlet' mode, use
+        the ``frequencies`` parameter instead.
+    frequencies : list of float | list of lists of float
+        The frequencies to compute the source power for. If you want to compute
+        the average power for multiple frequency bins, specify a list of
+        lists: each list containing the frequencies for the corresponding bin.
+        Only used in 'cwt_morlet' mode. In other modes, use the ``freq_bins``
+        parameter instead.
+    noise_csds : list of instances of CrossSpectralDensity | None
+        Noise cross-spectral density for each frequency bin. If these are
+        specified, the DICS filters will be applied to both the signal and
+        noise CSDs. The source power estimates for each frequency bin will be
+        scaled by the estimated noise power (signal / noise).
```

Comment:
With `noise_csds=None` no noise CSDs will be used at all. Doing (signal / noise) is one way of normalizing the result. Alternatives are normalizing the leadfield or normalizing the filter weights. The parameters mostly exists to not break backwards compatibility.

In the new DICS implementation, the preferred way to do this is like this:

```python
csd_baseline = csd_morlet(epochs, tmin=-0.2, tmax=0, ...)
csd_signal = csd_morlet(epochs, tmin=0, tmax=1.0, ...)
filters = make_dics(info, fwd, csd_signal)  # make DICS filters on signal CSD
power_baseline = apply_dics(filters, csd_baseline)
power_signal = apply_dics(filters, csd_signal)

# Compute change in power relative to the baseline
power = power_signal / power_baseline
```

It makes it explicit what you're doing (and intuitively gives you options to do it differently) instead of hiding this in a `noise_csd` parameter.